app/uk/gov/hmrc/tariffclassificationfrontend/forms/CommodityCodeConstraints.scala

```diff
@@ -28,6 +28,7 @@ class CommodityCodeConstraints @Inject()(commodityCodeService: CommodityCodeServ
   private implicit val clock: Clock = appConfig.clock
 
   val commodityCodeExistsInUKTradeTariff: Constraint[String] = Constraint("constraints.commodityCodeExists")({
+    case s: String if s.isEmpty => Invalid("decision_form.error.bindingCommodityCode.required")
```

Comment:
Would this change the behaviour of liability forms in which we use this constraint after emptyOr?